src/bun.js/bindings/ErrorCode.cpp

```
@@ -201,14 +204,14 @@ WTF::String JSValueToStringSafe(JSC::JSGlobalObject* globalObject, JSValue arg)
         auto name = JSC::getCalculatedDisplayName(vm, cell->getObject());
         if (catchScope.exception()) {
             catchScope.clearException();
-            name = "Function"_s;
+            name = ""_s;
```

```suggestion
         return "[Function: (anonymous)]"_s;
```

---

I assuming that is fine to ignore this exception